tools/testing/source_code_test.py

```diff
@@ -91,6 +91,7 @@ def test_no_private_tf_api():
         "tensorflow_addons/optimizers/moving_average.py",
         "tensorflow_addons/metrics/r_square.py",
         "tensorflow_addons/utils/test_utils.py",
+        "tensorflow_addons/layers/adaptive_pooling.py",
```

Comment:
Please find an alternative. You can copy the code in tensorflow addons if you wish. Those blacklist are supposed to get smaller with the time. Rules are meant to be respected :)

Reply:
@Susmit-A you can directly copy codes into `tensorflow_addons/utils/keras_utils.py` directly :-)

Reply:
I've made the requested changes.